src/recensio/plone/subscribers/review.py

```diff
@@ -379,9 +379,9 @@ def generatePageImages(self, later=True):
 def review_pdf_updated_eventhandler(obj, evt):
     """Re-generate the pdf version of the review, then update the cover image
     of the pdf if necessary."""
-    if not obj.REQUEST.get("pdf_file"):
```

Comment:
I think the intention here is that if a PDF is uploaded for a review then we don't want to generate one. Of course this check doesn't work any more, but we may need some replacement.